internal/pkg/agent/install/user_windows.go

```diff
@@ -20,7 +19,16 @@ import (
 )
 
 const (
-	passwordLength = 127 // maximum length allowed by Windows
+	// Reference: https://learn.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements
+	passwordMinLength = 32 // Minimum length for better security
+	passwordMaxLength = 64 // Upper limit to avoid policy issues
```

Comment:
Do you have a reference for what issues could happen with too long of a password?  That would be a nice breadcrumb for our future selves if we have to change this.

Reply:
Why the 64 cap? Windows GUI allows up to 127 character length and AD allows 256 characters.